### ISSUE TITLE
Nested Routes raise error in production mode

### ISSUE THREAD

**Issue by giosava94** [wduckitt/React-Automation-Studio#21]
Hello,
I'm using V2.2.0 for both the example project and react-automation-studio. 
I have a set o nested route. Here an example code:
```
<BrowserRouter>
    <Switch>
        <Route path="/Example/Example3" component={Example3} />
        <Route path="/Example" component={Example} />
    </Switch>
</BrowserRouter>
```

(I tried this same code directly from the _Routes.js_ file in React-Automation-Studio)

When in development mode I can go to **http://localhost:3000/Example** and **http://localhost:3000/Example/Example3** with no problems. 
When in production mode I can go to **http://localhost:9000/Example** but **http://localhost:9000/Example/Example3** is not reachable when typing it in the url bar or when refreshing the page. It is reachable if I have a button redirecting to that page but again I can't refresh the page without errors.

The browser console raises the following warning:
- **The script from “http://localhost:9000/Example/static/js/2.a8a5fb70.chunk.js” was loaded even though its MIME type (“text/html”) is not a valid JavaScript MIME type.**

Moreover it raises the following errors:
- **The stylesheet http://localhost:9000/Example/static/css/2.ba36a308.chunk.css was not loaded because its MIME type, “text/html”, is not “text/css”.
- Uncaught SyntaxError: expected expression, got '<'**

The problem happens both with firefox and chrome.
The problem does not happen in a stand alone react project.

Had anyone have a similar problem?
I think there may be a problem with the static files in the build process.

**Comment by wduckitt** [comment]
Hi

Have a look at the react router https://reactrouter.com/web/api/Route

I think you must add in the exact or strict prop.

Regards,

William

**Comment by wduckitt** [comment]
Im not sure on this. Perhaps see if one of the props fix it,

I might have something to do with the  "homepage": "." in the package.json and also the version of the react-scripts package

I am busy with adding features in for release 3.0.0 can you try figure it out please. We dont currenly use nested routes so its not on my radar.

**Comment by giosava94** [comment]
Hi,

Those props are not related to this problem, in fact I tried adding the props but still the same error.
Anyway thanks for the response.

On the other hand I reminded that, in V2.2.0, you serve the react app from a node express server. At this link https://medium.com/@13milliseconds/nested-routes-rendering-with-react-router-and-express-is-not-working-2fbc70edf44d the author points out this problem.
In this article the author refers to a discussion in stack overflow where they suggest to add, to the **head** tag in the index.html file, the following tag: `<base href="/" />`. 
Moreover they explain in details from where the problem originates.

Doing so and rebuilding the project solved the problem.

I will open a pull request with the changes and close this issue.
Regards
